src/components/Section/Foreign/Activities/BenefitActivity/FutureBenefit.jsx

```diff
@@ -114,6 +114,8 @@ export default class FutureBenefit extends ValidationElement {
             className="begin"
             {...this.props.Begin}
             label={i18n.t('foreign.activities.benefit.future.label.begin')}
+            minDate={new Date()}
```

Comment:
This works for allowing futures dates but the minDateEqual to property is not working as expected here. 
I can set to 1 day in the future but can't set to today's date. @ryanhofdotgov

Reply:
Yeah, if it's not working as expected with minDateEqual we should pull it out for now and do it separately.